src/Components/HomePage/FeaturedBannersSlider.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import APIcall from './APIcall';
+import { Slider } from '../Slider/Slider';
+
+const Container = styled.section`
+    width: 100%;
+    height: 90vh;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    overflow: hidden;
+    margin-bottom: 3rem;
+`
+
+const FeaturedBannersSlider = () => {
+    const [FetchedData, setFetchedData] = React.useState(null);
+    const [error, setError] = React.useState(null)
+    const [isLoading, setIsLoading] = React.useState(false);
+
+    const URL = 'https://raw.githubusercontent.com/Herkom/wa-react-2021-q4-capstone-project/feat/deliverable1/mocks/en-us/featured-banners.json';
```

Comment:
You should be importing these files from your repository, as these are mock files provided for the project, not from the repository raw files in github.